### ISSUE TITLE
Unable to create advanced playlists.

### ISSUE THREAD

**Issue by blindndangerous** [AzuraCast/AzuraCast#2479]
**Using Docker installation method**

No
**Host Operating System**

Ubuntu 18.04.4
**Describe the bug**

When trying to make an advanced playlist, the edit field doesn't show up once the radio button is selected.  If I press sae changes anyway, I get an error that says your request could not be completed at this time.
**To Reproduce**

Try to make an advanced playlist from the playlists page. Select advanced as the type, and click save changes.  It should error.
**Expected behavior**

The playlist should be created and an edit field should come up at the bottom for editing liquidsoap's configuration.  I do know that you are working on redoing the way that's done at least by the edit configuration page, so am unsure if this will be fixed in a later update.
**Relevant Logs**

[2020-03-06T12:44:02.413509-07:00] AzuraCast.INFO: Object(App\Entity\StationPlay
list).type:     The value you selected is not a valid choice. (code 8e179f1b-97a
a-4560-a02f-2a8b42e49df7)  {"file":"/var/azuracast/www/src/Controller/Api/Abstra
ctApiCrudController.php","line":144,"code":0} []
**Screenshots**

**Device(s):**
 - Device:  
 - OS: 
 - Browser: 
 - Version: 

**Additional context**

**Comment by blindndangerous** [comment]
This is unrelated, but in your template you have here https://github.com/AzuraCast/AzuraCast/blob/master/SUPPORT.md
This just goes to a page where it says you've moved your help pages to https://www.azuracast.com/help/#submitting-an-issue
However, after clicking view logs and looking at the ansible instaallation paths, I don't have azuracast.log or a few others.  I guessed it was app.log now.  Guessing the page just wasn't updated.

**Comment by SlvrEagle23** [comment]
@blindndangerous My local tests aren't showing any problem with selecting advanced playlists, so I suspect this may be related to #2480, with the radio button not being properly selected.

I will get the support links updated accordingly.

**Comment by blindndangerous** [comment]
Thanks for updating the links.  If I select anything else, play x per songs, or hour, those choices come up just as normal.  But once advanced is selected and I go to just past the warning line about liquidsoap's internal configuration I just have submit, save changes and close at which point it errors.  Let me know if their's anything you need from me.

**Comment by blindndangerous** [comment]
I can now make playlists save when the advanced radio button is selected, however when going to the advanced tab, I cannot see any edit fields for editing the configuration.  Is this all done through the edit liquidsoap configuration link or does that get populated when the playlist is being edited?

**Comment by SlvrEagle23** [comment]
@blindndangerous Correct, this is done via the Edit Liquidsoap Configuration page.

**Comment by blindndangerous** [comment]
I'll close this for now as the problem seems to have been fixed. If something happens with this again, I'll bring this back up.  Thank you again for the work.
